static/templates/gear_menu.hbs

```diff
@@ -8,6 +8,30 @@
             view is a Bootstrap nav tab, even though we don't show the tab
             anymore --}}
             <li class="invisible" style="display:none;" role="presentation"><a href="#message_feed_container" data-toggle="tab"></a></li>
+            <li class="org-info org-name">{{realm_name}}</li>
+            <li class="org-info org-url">{{realm_uri}}</li>
+            <li class="org-info org-plan hidden-for-spectators">
+                {{#if is_plan_limited }}
+                    <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">Zulip Cloud Free</a>
```

Comment:
@alya should these plan name strings be tagged for translation? Since the landing pages aren't translated, it's a bit unclear to me whether this bit should be.

Reply:
Good question! As also noted below, maybe we should keep the plan names in English.